### PR TITLE
Add integration tests for switching the language

### PR DESCRIPTION
**Pull request by mukeshk**
Addressed to #214

### CODE REVIEW

File: src/test/robotframework/site/change-lang/logic.robot [hunk 1]
```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation   Verify change language scenario
+Library         SeleniumLibrary
+Suite Setup     Before Test Suite
+Suite Teardown  Close Browser
+Force Tags      site  change-lang  logic
```
Comment (php-coder) [inline src/test/robotframework/site/change-lang/logic.robot]: Let's rename "change-lang" to "lang".

File: src/test/robotframework/site/change-lang/logic.robot [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+*** Test Cases ***
+Verify home page logo text changes after language switch
+    Element Text Should Be  id=logo  My stamps
+    Click Link              //*[@href="?lang=ru"]
```
Comment (php-coder) [inline src/test/robotframework/site/change-lang/logic.robot]: Could you, please, add `id="change-lang-link"` to this element?

File: src/test/robotframework/site/change-lang/logic.robot (hunk 1 above)
Comment (php-coder) [reply src/test/robotframework/site/change-lang/logic.robot]: `site` -> `main-page` to be consistent with other tests in this directory.

File: src/test/robotframework/site/change-lang/logic.robot [hunk 3]
```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation   Verify change language scenario
+Library         SeleniumLibrary
+Suite Setup     Before Test Suite
+Suite Teardown  Close Browser
+Force Tags      site  change-lang  logic
+
+*** Test Cases ***
+Verify home page logo text changes after language switch
```
Comment (php-coder) [inline src/test/robotframework/site/change-lang/logic.robot]: `Language should be changed after switching language`

File: src/test/robotframework/site/change-lang/logic.robot [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    Element Text Should Be  id=logo  My stamps
+    Click Link              //*[@href="?lang=ru"]
+    Element Text Should Be  id=logo  Мои марки
```
Comment (php-coder) [inline src/test/robotframework/site/change-lang/logic.robot]: Could you use one tab for alignment of the keywords in the test case? (like you did in "Before Test Suite")

**Comment by mukeshk** [comment]
please review

**Comment by php-coder** [comment]
Thank you! Merged!

**Pull request merged**